packages/flutter/test/widgets/list_view_horizontal_test.dart

```diff
@@ -35,7 +35,7 @@ void main() {
     await tester.pumpWidget(buildFrame(textDirection: TextDirection.ltr));
 
     await tester.pump(const Duration(seconds: 1));
-    await tester.drag(find.text('1'), const Offset(-300.0, 0.0));
+    await tester.drag(find.text('1'), const Offset(-318.0, 0.0));
```

Comment:
This seems wrong. Nothing should have changed here: there's nothing competing with the drag gesture.